receiver/purefareceiver/README.md

```diff
@@ -55,15 +58,17 @@ receivers:
     env: dev
     settings:
       reload_intervals:
-        array: 10s
-        hosts: 13s
-        directories: 15s
-        pods: 30s
-        volumes: 25s
+        array: 20s
+        hosts: 60s
+        directories: 60s
+        pods: 60s
+        volumes: 60s
 
   purefa/array02:
     fa_array_name: foobar02
-    endpoint: http://127.0.0.1:9490/metrics
+    endpoint: https://127.0.0.1/metrics
```

Comment:
I wonder if it's possible to obtain the server CA in some way, and use it here. Something for a follow-up PR though.

Reply:
I don't think is quite necessary, as if the end user wanted the connection to be trusted they would import the CA into their root certs on their system to trust the array.